verification/curator-service/api/src/clients/email-client.ts

```diff
@@ -14,30 +14,21 @@ export enum EmailService {
  * SMTP client that sends email via nodemailer.
  */
 export default class EmailClient {
-    private constructor(
-        private readonly service: EmailService,
-        private readonly transport: Mail,
-    ) {}
+    private service?: EmailService;
+    private transport?: Mail;
+    constructor(private user?: string, private password?: string) {}
 
     /**
-     * Creates a client instance based on the provided user/pass,
+     * Initializes a client based on the stored user and password fields.
      *
-     * Both params must be provided to establish a real (Gmail-based)
+     * Both fields must be present to establish a real (Gmail-based)
      * connection. If either is absent, a testing service (Ethereal) is used.
-     *
-     * @param user - The email address to send mail from (e.g. foo@gmail.com).
-     * @param password - The password for the above account.
      */
-    static async create(
-        user?: string,
-        password?: string,
-    ): Promise<EmailClient> {
-        let service: EmailService;
-        let transport: Mail;
-        if (!user || !password) {
-            service = EmailService.Ethereal;
+    async initialize(): Promise<EmailClient> {
```

Comment:
do you want to store the fact that it was initialized and just return this if this gets called another time instead of creating new stuff?